src/main/java/com/metamx/metrics/JvmThreadsMonitor.java

```diff
@@ -0,0 +1,56 @@
+package com.metamx.metrics;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableMap;
+import com.metamx.emitter.service.ServiceEmitter;
+import com.metamx.emitter.service.ServiceMetricEvent;
+import java.lang.management.ManagementFactory;
+import java.lang.management.ThreadMXBean;
+import java.util.List;
+import java.util.Map;
+import org.gridkit.lab.jvm.perfdata.JStatData;
+import org.gridkit.lab.jvm.perfdata.JStatData.LongCounter;
+
+public class JvmThreadsMonitor extends AbstractMonitor
+{
+  private final Map<String, String[]> dimensions;
+
+  private int lastLiveThreads = 0;
+  private long lastStartedThreads = 0;
+
+  public JvmThreadsMonitor()
+  {
+    this(ImmutableMap.<String, String[]>of());
+  }
+
+  public JvmThreadsMonitor(Map<String, String[]> dimensions)
+  {
+    Preconditions.checkNotNull(dimensions);
+    this.dimensions = ImmutableMap.copyOf(dimensions);
+  }
+
+  @Override
+  public boolean doMonitor(ServiceEmitter emitter)
+  {
+    ThreadMXBean threadBean = ManagementFactory.getThreadMXBean();
```

Comment:
Also kept it for code consistency reason, because in other places where we use `ManagementFactory` we get Beans on each `doMonitor` call. And moreover `ThreadMXBean` itself is cached into `ManagementFactoryHelper`.